packages/epo-widget-lib/src/widgets/SourceSelector/Point/Point.tsx

```diff
@@ -51,11 +53,13 @@ const Point: FunctionComponent<PointProps> = ({
     return "transparent";
   };
 
-  const baseRadius = getRadius(id, radius);
+  const baseRadius = getRadius(type, radius);
   const activeRadius = Math.max(10, baseRadius * 1.2);
 
   return (
     <Styled.Point
+      data-id={id}
```

Comment:
assigning the source type and id to data attributes so we can figure out what was clicked back up at the top level